File: result/translate/chatgpt/python2java/java/CHECK_LARGE_NUMBER_DIVISIBLE_11_NOT.java
public static boolean f_gold(String str) {
    int n = str.length();
    int oddDigSum = 0;
    int evenDigSum = 0;
    for (int i = 0; i < n; i++) {
        if (i % 2 == 0) {
            oddDigSum = oddDigSum + (str.charAt(i) - '0');
        } else {
            evenDigSum = evenDigSum + (str.charAt(i) - '0');
        }
    }
    return ((oddDigSum - evenDigSum) % 11 == 0);
}

File: dataset/python2java/python/CHECK_LARGE_NUMBER_DIVISIBLE_11_NOT.py
def f_gold ( str ) :
    n = len ( str )
    oddDigSum = 0
    evenDigSum = 0
    for i in range ( 0 , n ) :
        if ( i % 2 == 0 ) :
            oddDigSum = oddDigSum + ( ord ( str [ i ] ) - 48 )
        else :
            evenDigSum = evenDigSum + ( ord ( str [ i ] ) - 48 )
    return ( ( oddDigSum - evenDigSum ) % 11 == 0 )